sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -25,6 +25,18 @@ export function useFormioDropdownLoader() {
     }));
   };
 
+  /**
+   * find the formio field and set the value
+   * @param form form.
+   * @param fieldId field api/id from formio.
+   * @param value value that need to be set to the field
+   */
+  const loadText = async (form: any, fieldId: string, value: any) => {
```

Comment:
It seems to be a method more suitable for the useFormioUtils.ts instead of useFormioDropdownLoader.ts.
Could you please move it?

Reply:
The others methods are using componentKey to identify the component, could you please rename fielded to componentKey?